### PR TITLE
fix(sources): updates typings definitions

### PR DESCRIPTION
**Pull request by lionelB**

### CODE REVIEW

File: packages/sources/index.js [hunk 1]
```diff
@@ -70,8 +72,10 @@ const getRouteBySource = (src) => routeBySource[src];
 
 // code-du-travail -> code_du_travail
 /** @type {cdtnSources.getSourceByRouteFn} */
-const getSourceByRoute = (slug) =>
-  sources.find((key) => routeBySource[key] === slug) || slug;
+const getSourceByRoute = (slug) => {
+  const [source] = routes.find(([, value]) => slug === value) || [];
+  return source ? source : null;
```
Comment (UnbearableBear) [inline packages/sources/index.js:77]: If think `source | null`is enough here but not sure

Comment (lionelB) [reply packages/sources/index.js:77]: if you mean `return source || null` ?   it's the same

**Review by UnbearableBear — APPROVED** [review]
If it's ok for you it's ok for me

File: packages/sources/index.js (hunk 1 above)
Comment (UnbearableBear) [reply packages/sources/index.js:77]: 👌

**Pull request merged**